widget/embedded/public/index.html

```diff
@@ -5,6 +5,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>Rango Widget</title>
     <link rel="stylesheet" href="index.css" />
+    <link
```

Comment:
It shouldn't be here. Please revert back `index.css` change and also remove `embedded/public` completely since it's not used.